packages/perspective-viewer-d3fc/src/js/plugin/template.js

```diff
@@ -13,7 +13,7 @@ import perspectiveStyle from "../../less/perspective-view.less";
 import template from "../../html/d3fc-chart.html";
 import {areArraysEqualSimple} from "../utils/utils";
 
-import {bindTemplate} from "@jpmorganchase/perspective-viewer/cjs/js/utils";
+import {bindTemplate} from "@jpmorganchase/perspective-viewer/src/js/utils";

```

Comment:
This is the cause of the build failure on Travis CI - the source file referenced here is ES2018.

Reply:
Thanks for the heads up! Fixed.